client/components/cocktail-list.jsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+
+class Cocktails extends React.Component {
+
+  constructor (props) {
+    super(props);
+    this.state = {
+      list: props.cocktails
+    };
+  }
+
+  _addCocktails () {
+    return this.props.cocktails.map(function (cocktail) {
+      return <li>{cocktail.name}</li>;
```

Comment:
Oh, and one more thing: React is asking for a unique key for each list item. (More on why it needs that [here](http://facebook.github.io/react/docs/multiple-components.html#dynamic-children)--basically, keys help in the diffing algo.) I bet your terminal is showing that complaint (plz provide keys) in the `npm run start` tab, but only after you hit the live route.

Luckily, we have convenient unique keys on hand!
```
return (
  <li key={cocktail.id}> // <=
    <a href={`cocktails/${cocktail.id}`}>{cocktail.name}</a>
  </li>
);
```